### PR TITLE
api: improve the output of HTTP API POST method

### PR DESCRIPTION
**Pull request by JmPotato**
Signed-off-by: JmPotato <ghzpotato@gmail.com>

### What problem does this PR solve?

Issue #2227  

### What is changed and how it works?

Replace all HTTP API POST/DELETE method‘s output with more specified messages.

### Check List

Tests

- Unit test
- Integration test

Code changes

- Has HTTP API interfaces change (Don't forget to [add the declarative for API](https://github.com/pingcap/pd/blob/master/docs/development.md#updating-api-documentation))

### CODE REVIEW

File: pkg/apiutil/apiutil.go [hunk 1]
```diff
@@ -117,13 +117,32 @@ func ReadJSONRespondError(rd *render.Render, w http.ResponseWriter, body io.Read
 func ErrorResp(rd *render.Render, w http.ResponseWriter, err error) {
 	if err == nil {
 		log.Error("nil is given to errorResp")
-		rd.JSON(w, http.StatusInternalServerError, "nil error")
+		rd.JSON(w, http.StatusInternalServerError, WrapOutput(http.StatusInternalServerError, "nil error"))
```
Comment (disksing) [inline pkg/apiutil/apiutil.go:120]: Maybe more easy to use (don't have to wrapOutput for every response) if we can somehow wrap the `render.Render`?

**Comment by JmPotato** [comment]
@disksing @rleungx PTAL

**Pull request merged**